source/ios/AdaptiveCards/ADCIOSVisualizer/resources/sample.json

```diff
@@ -246,7 +246,6 @@
 		"spacing": 5
 	},
 	"media": {
-		"playButton" :"https://www.shareicon.net/download/64x64/2017/04/08/883361_media_512x512.png",
```

Comment:
should we perhaps host a play button icon on adaptivecards.io that you can reference? seems like it's worth having a custom `playButton` in our tests...

Reply:
Alternatively can we get this image added as part of the test artifacts/resources that are referenced locally and dont require an external call over the wire? 

---
In reply to: [299165267](https://github.com/microsoft/AdaptiveCards/pull/3126#discussion_r299165267) [](ancestors = 299165267)

Reply:
only concern would be if there's a runtime difference between pulling from a URL vs. local disk :)

---
In reply to: [299171295](https://github.com/microsoft/AdaptiveCards/pull/3126#discussion_r299171295) [](ancestors = 299171295,299165267)

Reply:
both approaches look good to me. i can add resource resolver to make references to a local bundle. 

I think that Paul's concern is legitimate one. I think that there is a value in testing imaging loading through network as they are the most frequently used mode of transport.  

I will create an issue to track this issue and for discussion.
[https://github.com/microsoft/AdaptiveCards/issues/3127](https://github.com/microsoft/AdaptiveCards/issues/3127)


---
In reply to: [299206526](https://github.com/microsoft/AdaptiveCards/pull/3126#discussion_r299206526) [](ancestors = 299206526,299171295,299165267)